x-pack/plugins/ml/public/application/jobs/new_job/common/components/time_range_picker.tsx

```diff
@@ -85,21 +85,23 @@ export const TimeRangePicker: FC<Props> = ({ setTimeRange, timeRange }) => {
             />
           }
           endDateControl={
-            <EuiDatePicker
-              selected={endMoment}
-              onChange={handleChangeEnd}
-              startDate={startMoment}
-              endDate={endMoment}
-              aria-label={i18n.translate(
-                'xpack.ml.newJob.wizard.timeRangeStep.timeRangePicker.endDateLabel',
-                {
-                  defaultMessage: 'End date',
-                }
-              )}
-              showTimeSelect
-              dateFormat={dateFormat}
-              minDate={startMoment}
-            />
+            <div data-test-subj="mlJobWizardDatePickerRangeEndDate">
```

Comment:
https://eui.elastic.co/#/forms/date-picker supports `data-test-subj`, hence you do not need a wrapper.

Reply:
I tried it, following the docs per our zoom, but it did not work.  I'll try again though!

Reply:
![Screenshot 2024-04-04 at 10 38 42](https://github.com/elastic/kibana/assets/4858051/083939c1-7fe4-4465-89ac-a67f43962344)
@darnautov perhaps a longer timeout?

Reply:
FYI adding div wrapper also braked styles. 

<img width="549" alt="image" src="https://github.com/elastic/kibana/assets/5236598/86f4aa55-af5d-4d95-8c42-90ddc2b175b4">

Reply:
@wayneseymour you set a `data-test-subj` prop in the wrong place. It should be for `EuiDatePickerRange`, not `endDateControl`

Reply:
yeah I dont think either of those work, so I've switched it to a span

Reply:
no more span!
https://github.com/elastic/kibana/pull/179358/commits/1f9611a6705951a925efd2e0c0a6f40a8554921e